django/apps/config.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, app_name, app_module):
         # This value must be unique across a Django project.
         if not hasattr(self, 'label'):
             self.label = app_name.rpartition(".")[2]
+        if not self.label.isidentifier():
+            raise ImproperlyConfigured(
+                "The app label '%s' is not a valid Python identifier." % self.label
+            )
```

Comment:
I guess this should be commented with the explanation.

```# app label must be a valid Python identifier```

Reply:
It feels like the code is pretty self documenting already.

Reply:
I don't see any value in adding a comment that matches the error message.